include/circt/Dialect/FIRRTL/FIRRTLCanonicalization.td

```diff
@@ -74,7 +74,7 @@ def KnownWidth : Constraint<CPred<[{
   $0.getType().isa<FIRRTLBaseType>() &&
     !$0.getType().cast<FIRRTLBaseType>().getRecursiveTypeProperties().hasUninferredWidth
   }]>>;
-
+  
```

Comment:
Nit: unrelated change.

Mega-nit: particularly egregious as this _adds_ whitespace. 😨